src/features/events/models/EventDataModel.ts

```diff
@@ -34,18 +35,62 @@ export default class EventDataModel extends ModelBase {
     this._repo = new EventsRepo(env);
   }
 
+  getAvailParticipants(): number {
```

Comment:
I think the naming of methods in this class are not entirely consistent. Some methods called `getXParticipants()` (or similar) return an array of participants, while others return a number. I would distinguish the methods returning numbers from the ones returning actual participants, using one of these patterns:

A. `getXParticipants()` returns list, `getNumXParticipants()` returns count, OR
B. `getXParticipants()` returns list, `getXParticipantCount()` returns count

Reply:
Did you do anything about this, @rebecarubio?